eventiq_asb/manager.py

```diff
@@ -0,0 +1,56 @@
+from azure.core import exceptions
+from azure.servicebus.aio.management import ServiceBusAdministrationClient
+from azure.servicebus.management import CorrelationRuleFilter
+from eventiq.logging import LoggerMixin
+
+
+class ServiceBusManager(LoggerMixin):
+    def __init__(self, connection_string: str) -> None:
+        self._service = ServiceBusAdministrationClient.from_connection_string(
```

Comment:
Klasa `Service`, ma w sobie `_service`. Tutaj może nazwałbym to `.client` albo `.sb_client` i spróbował z tą klasą np `ServiceBusManagerMiddleware`

Reply:
done